### ISSUE TITLE
false positive on aws.Context

### ISSUE THREAD

**Issue by SVilgelm** [kkHAIKE/contextcheck#6]
Hi, we do use a lot the aws-sdk-go lib and all their functions use an alias of context: https://github.com/aws/aws-sdk-go/blob/main/aws/context_1_9.go#L12

so the contextcheck fails on the code that uses those functions:
```
aws/awsCaller.go:34:53: Function `GetCallerIdentityWithContext` should pass the context parameter (contextcheck)
        callerId, err = awsSTS.GetCallerIdentityWithContext(ctx, &sts.GetCallerIdentityInput{})
```

when
```go
func (c *STS) GetCallerIdentityWithContext(ctx aws.Context, input *GetCallerIdentityInput, opts ...request.Option) (*GetCallerIdentityOutput, error) {
	req, out := c.GetCallerIdentityRequest(input)
	req.SetContext(ctx)
	req.ApplyOptions(opts...)
	return out, req.Send()
}
```

Could you please add a check is the first parameter is an alias of the `context.Context`, like it is done in aws-sdk-go

**Comment by SVilgelm** [comment]
it is impossible to exclude all the functions from aws-sdk-go, because there are lot of them :)

**Comment by kkHAIKE** [comment]
have you tried the last version of golang-ci

**Comment by vitaly-eureka-security** [comment]
> have you tried the last version of golang-ci

Happens to me as well with golang-ci 1.49.0

**Comment by kkHAIKE** [comment]
fixed, wait next golang-ci release 😊

**Comment by SVilgelm** [comment]
thank you very much!!!
